src/main/java/duke/exception/InvalidDateException.java

```diff
@@ -0,0 +1,10 @@
+package duke.exception;
+
+/**
+ * Represents an exception thrown when date format is invalid.
+ */
+public class InvalidDateException extends DukeException {
+    public InvalidDateException() {
```

Comment:
Missing constructor comment? Same for your other 3 exceptions

Reply:
Yes, added